src/main/java/com/netgrif/application/engine/petrinet/domain/arcs/reference/Referencable.java

```diff
@@ -3,4 +3,6 @@
 public interface Referencable {
 
     int getMultiplicity();
+
+    Referencable clone();
```

Comment:
Why was this added? Cloning is not the concern of Referencable

Reply:
Because the Arc class contains only Referencable interface, and not the regarded implementations (Place, Datafield)